blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,166 @@
+
+This support has been instrumental in allowing PyMC to continue to grow and thrive in the Open Source space, as well as in expanding community and diversity efforts.
+
+### **2017-2020**
+
+In October 2017, the team behind Theano, a deep learning framework, decided to discontinue the project due to the availability of other well-supported frameworks. 
+
+This presented a challenge for PyMC, which relied heavily on Theano. The PyMC team spent over a year evaluating other computational backends, including MXNet, TensorFlow, and PyTorch, before deciding to try TensorFlow.  TensorFlow had been supportive of PyMC and was developing TensorFlow Probability, which included components that would be useful for PyMC. However, the rapid changes to TensorFlow presented many challenges.
+
+![Theano with JAX](../_static/chris-pymcon-keynote/Theano_with_Jax.png)
+
+In 2020, while working on the symbolic PyMC project, Brandon Willard had the idea to link Theano to [JAX](https://jax.readthedocs.io/en/latest/notebooks/quickstart.html) as a computational back end. This would allow Theano to take advantage of JAX's autograd and linear algebra acceleration capabilities without the constraints of a deep learning framework. Willard successfully developed a JAX linker for Theano and used it to create the Theano-pymc library, to be used as a back-end for PyMC.
+
+The first [PyMCon](https://pymc-devs.github.io/pymcon/speakers) took place in October 2020. Chris Fonnesbeck closes out the talk by thanking everyone who worked to make the conference a reality, including Executive Directors Thomas Wiecki and Ravin Kumar. You can view the presentations here, or view the full talks on [the PyMCon playlist on Youtube](https://www.youtube.com/watch?v=UznM_-_760Y&list=PLD1x-BW9UdeHN2vwR6kIApJATd2jZzeya).
+
+## Present Day
+
+![Golden Age](../_static/chris-pymcon-keynote/Present_day_golden_age.png)
+
+How far have we come from the early 2000s?
+
+As Chris Fonnesbeck says, we are currently in the “golden age of probabilistic programming” — there are  now many options to implement bayesian models on different platforms.
+
+Today, when you build a model in PyMC, the interface is designed to be intuitive and easy to use. PyMC aims to make it as easy to code your model as it is to write it down on a whiteboard:
+
+![Model in PyMC](../_static/chris-pymcon-keynote/PyMC_model_sample_code.png)
+
+You can simply run the [pm.sample](https://www.pymc.io/projects/docs/en/stable/api/generated/pymc.sample.html) function to fit your model, which automates many of the decisions involved in the process. This allows you to generate MCMC results with just one line of code, without having to worry about the technical details of the algorithm. Thomas Wiecki, one of the founding developers, refers to this as the "automatic inference button".
+
+## What’s Next?
+
+Want to get started with bayesian analysis? 
```

Comment:
I believe "Bayesian" analysis is usually capitalized.
Can search the internet to confirm.